.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -0,0 +1,34 @@
+---
+name: Bug report
+about: Create a report to help us improve the ohsome API.
+title: ''
+labels: 'bug'
+assignees: ''
+
+---
+
+### Bug Description
+A clear and concise description of what the bug is.
+
+### General Information
+Please include the following general information about the issue and list any additional steps needed to reproduce the bug.
+ - Version of the ohsome API [e.g. `1.4.0`]
+ - Which API instance was requested [e.g. `https://api.ohsome.org/v1`]
+ - Affected endpoint(s) [e.g. /contributions/count]
+ - URL of your request
```

Comment:
one last point: since we already asked for the api instance and endpoint, there's no need to ask for the URL here again (at least for POST requests). What we however need to know are the used parameters:

```suggestion
 - parameters used for your request
```

Reply:
I think it is useful to know the URL, at least it would be fast for us to just copy/paste and check the bug (if any). Errors could also come because of a bad request and not because of a bug, e.g. filter syntax used in the wrong way.
Instead of that, I would just change the text and get rid of URLs in the point about the API instance:
 - Which API instance was requested [e.g. remote instance, locale instance with an h2 file or to the cluster connected, ohsome dashboard, etc.]
 - URL of your request (and request body if applicable)

Reply:
sounds good